prefetch.bs

```diff
@@ -713,6 +714,18 @@ This section contains patches to [[NAVIGATION-TIMING]].
     Add an additional parameter to [=create the navigation timing entry=], which is a [=string=] <var ignore>deliveryType</var>, and pass it as an additional argument to [=setup the resource timing entry=].
 </div>
 
+<h2 id="interaction-with-other-specs">Interaction with other specifications and concepts</h2>
+
+<div algorithm="clear prefetch cache for origin patch">
+  Modify [$clear cache for origin$] to add the following step at the end.
+
+  1. [=list/For each=] [=navigable/top-level traversable=] |traversable| in the user agent's [=top-level traversable set=]:
+    1. Let |navigables| be the [=list=] of [=Document/inclusive descendant navigables=] from |traversable|'s [=navigable/active document=].
+    2. Let |documents| be the [=list=] of |navigables|' [=navigable/active document=]s.
+    3. [=list/For each=] [=navigable/active document=] |activeDocument| in |documents|:
```

Comment:
It's probably cleaner to loop over _navigables_ and then inside the loop look at `|navigable|'s [=navigable/active document=]`.